src/components/ec-input-field/ec-input-field.vue

```diff
@@ -156,6 +162,9 @@ const inputModel = computed<InputFieldProps['modelValue']>({
 const inputClasses = computed(() => {
   const classes = ['ec-input-field__input'];
 
+  if (props.bgGrayColorLevel) {
+    classes.push(`tw-bg-gray-${props.bgGrayColorLevel}`);
```

Comment:
this won't be recognised by CSS class cleaning tool as purgeCSS for example. We need to define classes for all levels in style tag and use them in this push. Otherwise we can loose this classes if they are not used in other cases in style tag or template, which is unlikely in this case as bg-gray is used widely, but....